src/Common/Common.Api/Gateways/Entities/MashupFunctionParameterValues.cs

```diff
@@ -0,0 +1,22 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System.Runtime.Serialization;
+
+namespace Microsoft.PowerBI.Common.Api.Gateways.Entities
+{
+    [DataContract]
+    public class MashupFunctionParameterValues
```

Comment:
sealed? just same as other class. Please check with other DataContract Class

Reply:
I would like to keep this the same as the backend code.